### PR TITLE
LPS-105631

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	| Container
+	| Fragment
+	| Fragment
+	| Root
+	| Row;
+
+export interface LayoutData {
+	items: {
+		[itemId: string]: LayoutDataItem;
+	};
```
Comment (jbalsas) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts]: Didn't we have here `version` and something else?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		paddingHorizontal: number;
+		paddingVertical: number;
+		type: 'fixed' | 'fluid';
+	};
+
+	type: 'container';
+}
+
+interface Fragment extends BaseLayoutDataItem {
+	config: {fragmentEntryLink: string};
```
Comment (jbalsas) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts]: Just out of curiosity... this is an `id`, but it's not `fragmentEntryLinkId`? I know this has tripped me in the past in this project...

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts]: That's a typo, it's a fragmentEntryLinkId

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/layoutData.d.ts]: Sure, I'm gonna add it

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/resolvedCommentsReducer.js [hunk 3]
```diff
@@ -12,39 +12,16 @@
...
-				}
-			}
-			break;
 		case TYPES.TOGGLE_SHOW_RESOLVED_COMMENTS:
-			nextState = {
-				...nextState,
-				showResolvedComments: action.showResolvedComments
-			};
-			break;
+			return action.showResolvedComments;
```
Comment (jbalsas) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/resolvedCommentsReducer.js]: Don't we need to spread `nextState` here?

Comment (jbalsas) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/resolvedCommentsReducer.js]: I guess we don't based on `combineReducers`... although I find it a bit confusing... 🤔

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/82763. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/5063faf7d539~1...5063faf7d539) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/82765. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/acca1978e352~1...acca1978e352) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**